src/i18n/strings/de_DE.json

```diff
@@ -287,6 +287,7 @@
     "File to import": "Zu importierende Datei",
     "Failed to invite the following users to the %(roomName)s room:": "Folgende Benutzer konnten nicht in den Raum \"%(roomName)s\" eingeladen werden:",
     "Are you sure you wish to remove (delete) this event? Note that if you delete a room name or topic change, it could undo the change.": "Bist du sicher, dass du dieses Ereignis entfernen (löschen) möchtest? Wenn du die Änderung eines Raum-Namens oder eines Raum-Themas löscht, kann dies dazu führen, dass die ursprüngliche Änderung rückgängig gemacht wird.",
+    "Reason (optional)": "Grund (optional)",
```

Comment:
translations don't get included here because they will cause merge conflicts. Please remove this one and let `yarn i18n` do its thing.